test/build.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable ava/no-ignored-test-files */
```

Comment:
Could you check with `npx xo` if this error still exists?

I remember this error only appears on the editor somehow.

Reply:
I see. You've added a new line `!test/build.js` to the `ava.files`.

Please ignore my comment if you are intentionally ignoring this file.

Reply:
Yeah, this test runs separate from the others. I guess the `test` script could be set up to build before running, so they could be together.